docs/deployment/landing_page.rst

```diff
@@ -384,3 +384,27 @@ SecureDrop submissions. We control the servers that store your submissions,
 so no third party has direct access to the metadata or content of what you send us.
 
 Do not discuss leaking or whistleblowing, even with trusted contacts.
+
+How to test your *Landing Page* using Tor Browser
+-------------------------------------------------
+
+*Sources* may visit your *Landing Page* using Tor.
+
+Many websites are configured with security measures that only apply
+when Tor is in use. For example, Tor visitors may be requested to solve
+a CAPTCHA, may trigger warnings that are specific to some Tor exit nodes,
+or may be unable to load the page altogether because of
+Tor-specific DDoS protections.
+
+The effect of such measures cannot be tested without using Tor, and it is
+a very bad experience for a *source* if visiting a *Landing Page* doesn't work
+as expected. Because of that, it is **strongly recommended** that you test
+your organization's *Landing Page* using Tor *before* you start advertising it.
+
+You can do it using Tor Browser:
+
+#. Download Tor Browser from the `Tor Project website`_.
+#. Visit your *Landing Page*.
+#. Verify that everything works as expected.
```

Comment:
I would add a point here about reloading the page using different Tor circuits to observe behavior across a range of exit nodes.

This can be done by clicking the padlock icon on the left side of the address bar, and clicking "New Circuit for this Site".

Reply:
Another thing worth testing is if the page loads correctly if the Tor Browser security settings are set to "Safest".

Reply:
I had a go at both, let me know if that's what you had in mind. : )